shared/utility/libraries.cmake

```diff
@@ -21,4 +21,13 @@ if(CMAKE_BUILD_TYPE STREQUAL "Debug")
   target_link_libraries(nuclear_utility PUBLIC libbacktrace::libbacktrace ${CMAKE_DL_LIBS})
 endif()
 
+# Generate the decoders for each of the message types we know
+add_custom_command(
+  OUTPUT "${source_file}"
```

Comment:
I am probably missing something, but `${source_file}` is undefined here, so I don't know what `add_custom_command` is meant to output

Reply:
I clearly copied the wrong add_custom_command.... this is what happens when you do too many PRs at the same time